test/testinput/010_VarField_britemp.yaml

```diff
@@ -125,3 +124,214 @@ observations:
     HofX: ObsValue # just a placeholder -- not used, but needed to force calls to postFilter.
     benchmarkFlag: 1000 # just to keep the ObsFilters test happy
     flaggedBenchmark: 0
+#
+# Now we test effect varChannels and compress var channels
+#
+  - obs space:
+      name: AMSUB
+      obsdatain:
+        engine:
+          type: H5File
+          obsfile: Data/010_VarField_britemp.nc4
+      simulated variables: [brightnessTemperature]
+      channels: 1,3
+    obs filters:
+    # Double all observation errors: we want to check if error changes made by filters are
+    # propagated to VarObs files
+    - filter: BlackList
+      action:
+        name: inflate error
+        inflation factor: 2.0
+    # Reject the first level of the first observation and the last level of the last observation
+    - filter: Bounds Check
+      minvalue: 1.15
+      maxvalue: 3.35
+    # Set the flag of observations with missing values to "pass": we want to check if these
+    # values are encoded correctly in the VarObsFile.
+    - filter: Reset Flags to Pass
+      flags_to_reset: [10, 15]  # missing, Hfailed
+    - filter: VarObs Writer
+      variables_for_quality_control:
+        - name: brightnessTemperature
+          channels: 1,3
+      namelist_directory: testinput/VarObsWriterNamelists_010_VarField_britemp
+      general_mode: debug
+      size_of_varobs_array: 3
```

Comment:
I was expecting this to be 2, since we are compressing to levels ["2","4"]? Or am I misunderstanding what compressing channels does?

Reply:
Why is there an empty slot? I assumed compressing removed the empty slots

Reply:
An empty slot ahead in this case, or you might need empty slots behind the values for example airs has dummy channel spaces. 
In this case it is applying the offsetchans effect, offsetting the channels by 1 which leaves the empty space at the start.